Tasks/PublishBuildArtifactsV1/Invoke-Robocopy.ps1

```diff
@@ -9,6 +9,9 @@ param(
     [Parameter(Mandatory = $true)]
     [int]$ParallelCount,
 
+    [Parameter(Mandatory = $true)]
```

Comment:
I don't think this needs to be Mandatory.

We can have it optional and default to "Normal" if it's not specified. We could also just make this a boolean value like to turn on and off logging verbose.

Reply:
Actually, I see that you have an option of Silent or Normal.

Are there only two options? If so then I think an On/Off boolean would be better.
If not, then the current solution is fine